Apps/Contoso.UWP.Puppet/App.xaml.cs

```diff
@@ -37,6 +37,11 @@ public App()
             Suspending += OnSuspending;
             AppCenter.LogLevel = LogLevel.Verbose;
             AppCenter.SetLogUrl("https://in-integration.dev.avalanch.es");
+            object storageSize;
```

Comment:
Can we set for this field the default value as in other places?
`var storageMaxSize = Settings.Default.StorageMaxSize;`

Reply:
There is no _Settings.Default_ for UWP.

Reply:
Can we use constant instead?

Reply:
These settings contains only objects. That is why the code was written in a such way.